library/data/tags/src/main/java/org/quiltmc/qsl/tag/api/TagType.java

```diff
@@ -21,33 +21,19 @@
  */
 public enum TagType {
 	/**
-	 * Represents a tag type whose tags are required to start but not to connect, they will sync to the client.
-	 */
-	SERVER_REQUIRED(true),
-	/**
-	 * Represents a tag type whose tags are required to start and to connect, they will sync if present on the server.
-	 * <p>
-	 * If the client has tags which are of this type, and the server does not,
-	 * the client will disconnect when attempting to connect to said server.
-	 */
-	REQUIRED(true),
-	/**
-	 * Represents an optional tag type, tags are not required to start nor to connect,
-	 * but they will sync if present on the server.
+	 * Represents the default tag type.
 	 * <p>
 	 * The client does not provide a default.
 	 */
-	OPTIONAL(false),
+	NORMAL,
 	/**
-	 * Represents a tag type whose tags are not required to start nor to connect,
-	 * but they will sync if present on the server.
-	 * <p>
-	 * The client provides a fallback in-case the server doesn't have the relevant tag.
+	 * Represents a tag type which is similar to the default one,
+	 * but the client provides a fallback in-case the server doesn't have the relevant tag.
 	 * <p>
 	 * If two tags with the same identifier are registered, one with this type and the other one with another,
 	 * they may not have the same content as the fallback content will only be added to the one with this type.
```

Comment:
Why is it even legal to register two tags with the same identifier but different types?

Reply:
I might need to change the way this is formulated since nothing is actually registered anymore.

But for a TagKey, the type is part of the key itself, which allows a tag with the same identifier to be used alongside a tag with another type.

Reply:
I have reformulated the tag types.